_pages/getting-started/classes/meetings-and-meeting-tools.md

```diff
@@ -8,11 +8,16 @@ Here, you'll find a list of tools folks at TTS use to schedule meetings, along w
 
 Here are some of the tools TTS uses to facilitate meetings:
 
-- [Google Meet]({{site.baseurl}}/google-meet/)
-- [Google Calendar]({{site.baseurl}}/google-calendar/)
-- [Zoom]({{site.baseurl}}/zoom/)
-- [Meeting Space]({{site.baseurl}}/gsa-internal-tools/#meeting-space) (for conference lines)
-- Adobe Connect
+For scheduling, use [Google Calendar]({{site.baseurl}}/google-calendar/). For videoconferencing:
```

Comment:
I think making this a table makes it very hard to read in the `md` and is highly likely to break responsiveness. Why not just a simple list? Would that work?

Reply:
I think a table is easier to understand at a glance on the site itself (especially when folks are doing comparison), but it's a tradeoff, yeah. Feel free to change back if you feel strongly.